contrib/epee/src/mlog.cpp

```diff
@@ -55,6 +55,7 @@ static std::string generate_log_filename(const char *base)
     strcpy(tmp, "unknown");
   else
     strftime(tmp, sizeof(tmp), "%Y-%m-%d-%H-%M-%S", &tm);
+  tmp[sizeof(tmp) - 1] = 0;
```

Comment:
I think strcpy and strftime both null-terminate the buffer. What is the point of setting the last byte to 0 here?